core/src/CabanaCore_config.hpp.cmakein

```diff
@@ -7,12 +7,4 @@
 
 #cmakedefine Cabana_ENABLE_MPI
 
-#cmakedefine Cabana_ENABLE_Serial
-
-#cmakedefine Cabana_ENABLE_Pthread
-
-#cmakedefine Cabana_ENABLE_OpenMP
-
-#cmakedefine Cabana_ENABLE_Cuda
```

Comment:
This might break code downstream silently.  Just making sure you realize that.

Reply:
Yes I agree this is a backwards compatibility change for the upcoming 1.0 release - documenting this in the PR notes as well.